cmd/nerdctl/builder_build_test.go

```diff
@@ -498,3 +498,50 @@ func TestBuildNetworkShellCompletion(t *testing.T) {
 	networkName := "default"
 	base.Cmd(gsc, "build", "--network", "").AssertOutContains(networkName)
 }
+
+func TestBuildAttestation(t *testing.T) {
+	testutil.DockerIncompatible(t) // ref:https://github.com/docker/buildx/issues/1655
+
+	t.Parallel()
+	testutil.RequiresBuild(t)
+	base := testutil.NewBase(t)
+	if testutil.GetTarget() == testutil.Docker {
+		base.Env = append(base.Env, "DOCKER_BUILDKIT=1")
```

Comment:
Do we still need this?

Reply:
Thanks @AkihiroSuda 
It has been remove.